recipe/meta.yaml

```diff
@@ -19,7 +19,7 @@ build:
   rpaths:
     - lib/R/lib/
     - lib/
-  skip: True  # [win]
+  skip: True  # [win32]
```

Comment:
When making this sort of change, one need to [re-render]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) the feedstock.